qiskit_ibm_runtime/ibm_runtime_service.py

```diff
@@ -117,10 +127,7 @@ class IBMRuntimeService:
     """
 
     def __init__(
-            self,
-            token: Optional[str] = None,
-            url: Optional[str] = None,
-            **kwargs: Any
+        self, token: Optional[str] = None, url: Optional[str] = None, **kwargs: Any
```

Comment:
I am not particularly fond of black doing this when it’s clearer on separate lines. Is there a setting to disable this?

Reply:
As Black is an opinionated code formatter, it does not leave much room for configuration.  You can read more about how Black wraps lines at https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#how-black-wraps-lines. In this case all arguments fit in a single line (<88 chars) and have therefore been updated.

As a final fallback we can always make use of 
>  It [Black] doesn’t reformat blocks that start with # fmt: off and end with # fmt: on, or lines that ends with # fmt: skip. 

In this case I personally would prefer to stick with the defaults.